src/parser/monk/mistweaver/modules/spells/Celestial.js

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import SPELLS from 'common/SPELLS';
+
+import Analyzer from 'parser/core/Analyzer';
+import HealingDone from 'parser/shared/modules/throughput/HealingDone';
+
+import SpellIcon from 'common/SpellIcon';
+import StatisticBox, { STATISTIC_ORDER } from 'interface/others/StatisticBox';
+
+import { formatNumber } from 'common/format';
+import { TooltipElement } from 'common/Tooltip';
+
+const debug = false;
+
+class Celestial extends Analyzer {
+  static dependencies = {
+    healingDone: HealingDone,
```

Comment:
I don't believe you need this.

Reply:
Wasn't sure if I'd need it if I wanted to figure out some stat about the percentage of how much Yu'Lon or Chi-ji contributed to total healing.